__tests__/topics.test.js

```diff
@@ -0,0 +1,32 @@
+const app = require('../app');
+const request = require('supertest');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+
+const {topicData, articleData, commentData, userData} = require('../db/data/test-data/');
+
+beforeEach(() => seed( {topicData, articleData, commentData, userData} ));
+afterAll(() => db.end());
+
+describe("GET: /api/topics", () => {
+    test("200: Responds with an array of topics with the correct keys", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body : { topics } }) => {
+            expect(topics.length).toBe(3);
```

Comment:
Very small detail, but as you are iterating `topics` the variable name `topic` instead of `item` might be better. No biggie 😄